source/class/qx/tool/compiler/targets/meta/Browserify.js

```diff
@@ -158,12 +158,17 @@ qx.Class.define("qx.tool.compiler.targets.meta.Browserify", {
       builtins.process = builtins._process;
 
       return new Promise((resolve, reject) => {
-        let b = browserify([], {
+        const options = {
           builtins: builtins,
           ignoreMissing: true,
           insertGlobals: true,
           detectGlobals: true
-        });
+        };
+        qx.lang.Object.mergeWith(
+          options,
+          this.getAppMeta().getAnalyser().getBrowserifyConfig()?.options || {}
```

Comment:
This appears to allow the user-provided options to override the default ones. I believe the default ones are mandatory to accomplish the goals of this feature.

Reply:
Fixed